js/views/modals/orderDetail/FulfillOrder.js

```diff
@@ -98,13 +98,19 @@ export default class extends BaseVw {
   }
 
   render() {
+    const cryptoDelivery = this.model.get('cryptocurrencyDelivery');
+    const transactionIDLength = cryptoDelivery && cryptoDelivery.max.transactionIDLength || 0;
+
     loadTemplate('modals/orderDetail/fulfillOrder.html', (t) => {
       this.$el.html(t({
         contractType: this.contractType,
         isLocalPickup: this.isLocalPickup,
         ...this.model.toJSON(),
         errors: this.model.validationError || {},
         fulfillingOrder: fulfillingOrder(this.model.id),
+        max: {
```

Comment:
Could we just future proof it and pass the whole object in? for example:

```
constraints: cryptoDelivery && cryptoDelivery.constraints || {},
```

Reply:
Makes sense to me. Done.